src/capture/activated/mod.rs

```diff
@@ -199,6 +201,28 @@ impl<T: Activated + ?Sized> Capture<T> {
         PacketIter::new(self, codec)
     }
 
+    pub fn for_each<F>(&mut self, handler: F)
+    where
+        F: FnMut(Packet),
+    {
+        let mut handler = Handler {
+            func: AssertUnwindSafe(handler),
+            panic_payload: None,
+            handle: self.handle,
+        };
+        unsafe {
+            raw::pcap_loop(
+                self.handle.as_ptr(),
+                -1,
+                Handler::<F>::callback,
+                &mut handler as *mut Handler<AssertUnwindSafe<F>> as *mut u8,
```

Comment:
I recently discovered that rust will most likely also be happy with `&mut handler as *mut _ as _`

Reply:
I don't like `as` because it does too many things and has some strange cases like `&mut T as *const T`, but `as` is the only way to do this conversion with this project's MSRV so I'm doing this as explicitly as possible to reassure myself of exactly what `as` is doing here.